lib/phoenix_component/declarative.ex

```diff
@@ -817,21 +817,20 @@ defmodule Phoenix.Component.Declarative do
   defp build_attrs_docs(attrs) do
     [
       "## Attributes\n",
-      for attr <- attrs, attr.doc != false, attr.type != :global, into: [] do
-        [
-          "\n* ",
-          build_attr_name(attr),
-          build_attr_type(attr),
-          build_attr_required(attr),
-          build_hyphen(attr),
-          build_attr_doc_and_default(attr, "  "),
-          build_attr_values_or_examples(attr)
-        ]
-      end,
-      if Enum.any?(attrs, &(&1.type == :global)) do
-        "\n* Global attributes are accepted."
-      else
-        ""
+      for attr <- attrs, attr.doc != false, into: [] do
+        if attr.type == :global do
+          build_attr_doc_and_default(attr, "  ")
+        else
+          [
+            "\n* ",
+            build_attr_name(attr),
+            build_attr_type(attr),
+            build_attr_required(attr),
+            build_hyphen(attr),
+            build_attr_doc_and_default(attr, "  "),
+            build_attr_values_or_examples(attr)
+          ]
```

Comment:
The change is that we now call the function that is expecting to be called, but in the case of global, only that function. The `Global attributes are accepted` message has been moved into it to simplify.